### PR TITLE
REL: initial build 4022.32 for XRTM1

### PR DESCRIPTION
**Pull request by nrwslac**
Initial build for XRTM1 PLC. Built for 4022.32, No build errors.

### CODE REVIEW

File: lcls-plc-xrt-optics/lcls-plc-xrt-optics/_Config/NC/Axes/M2L0-Bender.xti [hunk 1]
```diff
@@ -0,0 +1,1535 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
Comment (ZLLentz) [inline lcls-plc-xrt-optics/lcls-plc-xrt-optics/_Config/NC/Axes/M2L0-Bender.xti:1]: Looks like you've got some lingering files from the LFE project in this repo

File: lcls-plc-xrt-optics/lcls-plc-xrt-optics/Untitled1/Untitled1.plcproj [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <FileVersion>1.0.0.0</FileVersion>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectGuid>{c87d048b-8e3d-4370-bd8e-5be7fa1b5c3a}</ProjectGuid>
+    <SubObjectsSortedByName>True</SubObjectsSortedByName>
+    <Name>Untitled1</Name>
```
Comment (ZLLentz) [inline lcls-plc-xrt-optics/lcls-plc-xrt-optics/Untitled1/Untitled1.plcproj:7]: It's probably worth renaming this at some point

Comment (ZLLentz) [reply lcls-plc-xrt-optics/lcls-plc-xrt-optics/Untitled1/Untitled1.plcproj:7]: I realized later in the review that this Unitled1 is an extra sub-project in this repo

**Review by ZLLentz — APPROVED** [review]
Everything looks good/normal to me, just missing the IO configuration and then we could theoretically move the real motors around.

I pointed out a couple places where lingering files from the LFE project and elsewhere were present

**Comment by ZLLentz** [comment]
One more thing to think about (outside of this PR), is how the PMPS should be handled from the XRT PLC? I'm not sure how this is set up because we're only using the Arbiter, etc. in the FEE and in TMO right now @slacAWallace

**Comment by ZLLentz** [comment]
Oh, there might be some syntax errors (or the parser is bugging out) based on the automatic testing thing: https://travis-ci.com/github/pcdshub/lcls-plc-xrt-optics/jobs/486093487#L477-L569

File: lcls-plc-xrt-optics/lcls-plc-xrt-optics/Untitled1/Untitled1.plcproj (hunk 2 above)
Comment (nrwslac) [reply lcls-plc-xrt-optics/lcls-plc-xrt-optics/Untitled1/Untitled1.plcproj:7]: I noticed this as well. I don't see this Untitled1 project in the project when I open it.

File: lcls-plc-xrt-optics/lcls-plc-xrt-optics/_Config/NC/Axes/M2L0-Bender.xti (hunk 1 above)
Comment (nrwslac) [reply lcls-plc-xrt-optics/lcls-plc-xrt-optics/_Config/NC/Axes/M2L0-Bender.xti]: removed

**Comment by nrwslac** [comment]
> One more thing to think about (outside of this PR), is how the PMPS should be handled from the XRT PLC? I'm not sure how this is set up because we're only using the Arbiter, etc. in the FEE and in TMO right now @slacAWallace

Definitely need some supervision on the PMPS system here. Especially when the older XRT HOMs code starts getting put in for XRTM2 and XRTM3.

**Comment by ghalym** [comment]
> One more thing to think about (outside of this PR), is how the PMPS should be handled from the XRT PLC? I'm not sure how this is set up because we're only using the Arbiter, etc. in the FEE and in TMO right now @slacAWallace

We are installing a new DRL in the Fee Alcove with the EL6695 to talk to the arbiter and  should connect back to the XRT PLC via Fiber.

**Comment by ZLLentz** [comment]
> We are installing a new DRL in the Fee Alcove with the EL6695 to talk to the arbiter and should connect back to the XRT PLC via Fiber.

That's really neat, I like it

**Pull request merged**